.gitmodules

```diff
@@ -22,3 +22,7 @@
 	path = third-party/miniupnpc
 	url = https://github.com/transmission/miniupnpc
 	branch = post-2.0.20170509-transmission
+[submodule "third-party/units"]
```

Comment:
Not convinced the amount and type of operations we do on unit-based variables warrants this dependency.